### ISSUE TITLE
Locked cache not mapped by MemoryMap

### ISSUE THREAD

**Issue by Kinnay** [decaf-emu/decaf-emu#457]
This is causing a crash in The Cave in a memset following an LCAlloc.

To me it seems like MemoryMap::mapMemory is simply missing the code to map the locked cache, as adding another else if statement with the proper variables seems to have fixed it.

Logs:
```
[51:10.466108 error:1704] Invalid physicalMemoryType 3 for mapMemory
[51:10.466797 error:1704] Unexpected failure allocating mapping virtual address 0xFFC00000 to physical address 0x02000000
```

**Comment by exjam** [comment]
Yeah looks like I missed that, you gonna PR that or you want me to push that fix?

**Comment by Kinnay** [comment]
https://github.com/decaf-emu/decaf-emu/pull/458

**Comment by exjam** [comment]
Thanks, good catch!

Merged as dd639d97852bb757c4d7097e15a11ebb1f5b7c42
